src/middleware/featureFlag.ts

```diff
@@ -4,6 +4,16 @@ import logger from "@root/logger/logger"
 import { RequestHandlerWithGrowthbook } from "@root/types"
 import { getNewGrowthbookInstance } from "@root/utils/growthbook-utils"
 
+// Keep one GrowthBook instance at module level
+// The instance will handle internal cache refreshes via a SSE connection
```

Comment:
not a problem with this PR but what does SSE mean?

Reply:
went to read the docs, this refers to server sided events. i assume the gb server issues events out to our client here in real time